docs/04-operation-guides/operations/smsh-01-istio-disable-sidecar-injection.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Disable Istio Sidecar Proxy Injection
+title: Disable Istio sidecar proxy injection
 ---
```

Comment:
Line 9: please fix the formatting of the command

Reply:
- As far as I notice we don't tend to capitalize all the words in a title, do we?
- I moved the command to another line.